vm-setup/azp_vm_setup.ps1

```diff
@@ -204,14 +214,16 @@ if ($Mode -eq "prepareseed") {
     }
     AddUnattendFile $Vhd "$($UnattendLocation)\unattend.xml"
     ExecuteScriptWithMountedVhd $Vhd {
-        Invoke-WebRequest -Uri "https://vstsagentpackage.azureedge.net/agent/2.172.2/vsts-agent-win-x64-2.172.2.zip" -OutFile "$($vhdmountdir)\vsts-agent-win-x64-2.172.2.zip"
+        Invoke-WebRequest -Uri "https://vstsagentpackage.azureedge.net/agent/2.192.0/vsts-agent-win-x64-2.192.0.zip" -OutFile "$($vhdmountdir)\vsts-agent-win-x64-2.192.0.zip"
         Invoke-WebRequest -Uri "https://github.com/OpenCppCoverage/OpenCppCoverage/releases/download/release-0.9.9.0/OpenCppCoverageSetup-x64-0.9.9.0.exe" -OutFile "$($vhdmountdir)\OpenCppCoverageSetup-x64-0.9.9.0.exe"
         Expand-Archive -Path "$($vhdmountdir)\vsts-agent-win-x64-2.172.2.zip" -DestinationPath "$($vhdmountdir)\a" -Force
         Copy-Item "vs_buildtools.exe" "$($vhdmountdir)\vs_buildtools.exe"
         Copy-Item "sfpcopy.exe" "$($vhdmountdir)\Windows\System32\sfpcopy.exe"
         Copy-Item "testroot-sha2.cer" "$($vhdmountdir)"
+        Copy-Item "dswdevice.exe" "$($vhdmountdir)"
+        Copy-Item "notmyfault64.exe" "$($vhdmountdir)"

```

Comment:
Is notmyfault64 compatible with x86/ARM?

Reply:
Not that I know of, but currently, we only test on amd64 machines.

Reply:
There is a 32 bit version, and theoretically the 64 bit version works on arm. I think Microsoft owns the source code, so we could likely find the owners and get an arm build if necessary

Reply:
Ah, yeah I see plenty of x64-only binaries here. Looks good to me.